src/shell.ts

```diff
@@ -537,8 +538,9 @@ export class Shell {
                 this.t.writeln("Invalid OTP, please try again")
         }
     }
-    async verifyFP(fp: string, prompt: string) {
-        let validated = false
+    async verifyFP(fp: string, prompt?: string) {
+        return terminal7.pb.verifyFP(fp, prompt)
```

Comment:
This function does nothing but calls terminal7.pb.verifyFP, better to remove it and call directly